packages/mc-scripts/src/commands/config:sync.js

```diff
@@ -0,0 +1,138 @@
+    );
+  }
+
+  const token = credentialsStorage.getToken(mcApiUrl);
+  const fetchedCustomApplication = await fetchCustomApplication({
+    mcApiUrl,
+    token,
+    entryPointUriPath: localCustomAppData.entryPointUriPath,
+  });
+
+  const userOrganizations = await fetchUserOrganizations({ mcApiUrl, token });
+
+  let organizationId, organizationName;
+
+  if (userOrganizations.total > 1) {
+    const organizationChoices = userOrganizations.results.map(
+      (organization) => ({
+        title: organization.name,
+        value: organization.id,
+      })
+    );
+
+    organizationId = await prompts({
+      type: 'select',
+      name: 'organizationId',
+      message: 'Select Organization',
+      choices: organizationChoices,
+      initial: 0,
+    })['organizationId'];
+
+    organizationName = organizationChoices.find(
+      ({ value }) => value === organizationId
+    ).title;
+  } else {
+    organizationId = userOrganizations.results[0].id;
+    organizationName = userOrganizations.results[0].name;
+  }
+
+  if (!fetchedCustomApplication) {
+    const { confirmation } = await prompts({
+      type: 'text',
+      name: 'confirmation',
+      message: `You are about to create the configuration in the "${organizationName}" organization in environment ${location}. Are you sure you want to proceed?`,
+      initial: 'yes',
+    });
+    if (confirmation.toLowerCase().charAt(0) !== 'y') {
+      console.log('Aborted.');
+    } else {
+      const createdCustomApplication = await createCustomApplication({
+        mcApiUrl,
+        token,
+        organizationId,
+        data: omit(localCustomAppData, ['id']),
+      });
+      // update applicationID in the custom-application-config file
+      updateApplicationIdInCustomApplicationConfig(createdCustomApplication.id);
+      const customAppLink = getMcUrlLink(
+        mcApiUrl,
+        organizationId,
+        createdCustomApplication.id
+      );
+      console.log(
+        chalk.green(
+          `You have successfully created the configuration in the "${organizationName}" organization in environment ${location}.\nYour local configuration has also been updated with the newly created application identifier: ${createdCustomApplication.id}.\nLink to the created custom application is ${customAppLink}`
+        )
+      );
+    }
+    return;
+  }
+  // update applicationID in the custom-application-config file
+  updateApplicationIdInCustomApplicationConfig(
+    fetchedCustomApplication.application.id
+  );
+
+  // TODO: show diff (followup task)
+
+  const { confirmation } = await prompts({
+    type: 'text',
+    name: 'confirmation',
+    message: `You are about to update the configuration in the "${organizationName}" organization in environment ${location}. Are you sure you want to proceed?`,
+    initial: 'yes',
+  });
+  if (confirmation.toLowerCase().charAt(0) !== 'y') {
+    console.log('Aborted.');
```

Comment:
Perhaps we could use `chalk` (_red_?) here to be more explicit with the message.